Vision/Vision.bs

```diff
@@ -38,9 +38,9 @@ Markup Shorthands: markdown yes
 
 	* Help the world understand what W3C is, what it does, and why it matters
 	* Communicate shared values and principles of the W3C community
-	* Be opinionated enough to provide a framework for making decisions,
+	* Be opinionated enough to provide guidance and a framework for making decisions,
 		particularly on controversial issues
-	* Be timeless enough to guide W3C yet flexible enough to evolve when needed
+	* Be timeless enough to remain relevant without needing frequent revision
```

Comment:
```suggestion
	* Be timeless enough to remain relevant without needing frequent revision, but evolve based on the needs of the community.
```

Reply:
(or possibly "be ready to evolve...".

Reply:
Thanks @cwilso. This is not exactly what I had in mind, but this (or the "be ready to evolve" variant) may be a reasonable compromise. I still find it a little off that it's in that section, but there's enough of a conceptual separation that it no longer hinders the original message.

I could live with this.

Reply:
Committed

> Be timeless enough to remain relevant without needing frequent revision,
> while being open to evolution based on the needs of the community

@cwilso Does that seem to capture what you're aiming for?

Reply:
Yes, although I would keep the verb "evolve" rather than the noun "evolution" to 1) mirror the verb "remain" in the first clause, and 2) avoid being mistaken for "revolution".

Reply:
OK. Changed to "evolving", though, because afaict "evolve" is the wrong tense there...